app/controllers/api/services_controller.rb

```diff
@@ -34,8 +34,7 @@ def settings
 
   def create
     @service = collection.new # this is done in 2 steps so that the account_id is in place as preffix_key relies on it
-    @service.attributes = params[:service]
-    @service.system_name = params[:service][:system_name]
+    @service.attributes = service_params
```

Comment:
only do this is the account is authorized to do it :smile: or if we are not doing the authorizations in depth yet, then at leaaast make sure the rolling update is enabled